pi5/config.go

```diff
@@ -55,15 +41,8 @@ type ConfigConverter = func(resource.Config, logging.Logger) (*LinuxBoardConfig,
 // BeagleBone or Jetson boards.
 func ConstPinDefs(gpioMappings map[string]GPIOBoardMapping) ConfigConverter {
 	return func(conf resource.Config, logger logging.Logger) (*LinuxBoardConfig, error) {
-		newConf, err := resource.NativeConfig[*Config](conf)
-		if err != nil {
-			return nil, err
-		}
-
 		return &LinuxBoardConfig{
-			AnalogReaders:     newConf.AnalogReaders,
-			DigitalInterrupts: newConf.DigitalInterrupts,
-			GpioMappings:      gpioMappings,
+			GpioMappings: gpioMappings,
```

Comment:
is 'GpioMappings' supposed to be gpioMappings (camelCase)?

Reply:
We capitalize it here to make it publicly accessible